<launch>
    <node pkg="markerlocator_odometry" name="markerlocator_odometry" type="markerlocator_odometry_node.py" output="screen">
        <param name="host" value="10.115.253.233" />
        <param name="port" value="21212" />
        <param name="frame_id" value="base_link" />
        <param name="covariance_x" value="0.001" />
        <param name="covariance_y" value="0.001" />
        <param name="covariance_z" value="0.001" />
        <param name="covariance_rot" value="99999" />
    </node>
</launch>
